apps/src/templates/SchoolAutocompleteDropdown.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component, PropTypes } from 'react';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
+import i18n from "@cdo/locale";
+import color from "../util/color";
+
+const styles = {
+  question: {
+    fontSize: 16,
+    fontFamily: '"Gotham 3r", sans-serif',
+    color: color.charcoal,
+    paddingTop: 10,
+    paddingBottom: 5
+  },
+  asterisk: {
+    fontSize: 20,
+    fontFamily: '"Gotham 5r", sans-serif',
+    color: color.red,
+  },
+  errors: {
+    fontSize: 14,
+    fontFamily: '"Gotham 3r", sans-serif',
+    color: color.red,
+    paddingTop: 5,
+    paddingBottom: 5
+  },
+};
+
+export default class SchoolAutocompleteDropdown extends Component {
+  static propTypes = {
+    setField: PropTypes.func,
+    showErrorMsg: PropTypes.bool,
+    // Value is the NCES id of the school
+    value: PropTypes.string
+  };
+
+  getOptions(q) {
```

Comment:
This method should also have some unit tests.  Fortunately it's got no dependencies on the rest of the component! :+1: